admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,55 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image. These checks are a set of diagnostic tasks meant to be run against a running OpenShift cluster.
+They are used to ensure Etcd cluster storage sanity, that correct package versions exist on the host machine, and to ensure a working logging stack deployment.
+
+
+An action plugin for the OpenShift Health Checks role automatically discovers all diagnostic checks, and executes only those selected in a play.
+Some checks have tags, which allow for groups of related checks to be run together as part of a play.
+
+The following table describes available health checks that can be run on a deployed OpenShift cluster.
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose |Tags
+
+|`*ovs_version*`
+|This check ensures that a host has the correct version of Open vSwitch installed for the currently deployed version of OpenShift.
+|`health`
+
+|`*kibana*`, `*curator*`, `*elasticsearch*`, `*fluentd*`
+|This set of checks verifies that `Elasticsearch`, `Fluentd`, and `Curator` pods have been deployed and are in a `running` state, and that a connection can be established between the control host and the exposed `Kibana` URL.
```

Comment:
Probably worth mentioning that it's only enabled if the logging deploy variable is set, so they don't expect it if they haven't used ansible to deploy the logging stack.